packages/acs-ui-common/src/common.ts

```diff
@@ -52,7 +52,7 @@ export function _getKeys<T extends object>(obj: T): (keyof T)[] {
  * a name that represents the name of file, and
  * a URL to download the attachment.
  *
- * @beta
+ * @public
```

Comment:
There should be changes to the stable API .md, is there a feature flag that needs to be added to the stabilized section?

Reply:
remove the cc flag in index.js to separate export, now it shows up, good catch thank you!